main.js

```diff
@@ -104,11 +104,11 @@ adapter.on('stateChange', (id, state) => {
         }
         return;
     }
-
-    let nodeId = parseInt(idParts[0].split('-')[1], 10);
-    if (nodeId === 0) nodeId = -1;
+	
+    let nodeId = parseInt(idParts[0].split('-')[1], 10);    
     const attributeId = parseInt(idParts[1].split('-')[1], 10);
-    const lookupId = nodeId + '.' + attributeId;
+    const lookupId =  nodeId + '.' + attributeId;
+	if (nodeId === 0) nodeId = -1;
```

Comment:
Please fix the two too much spaces :)

Reply:
done, seems tab-stops aren't the way to go here..